packages/cli/src/util/deploy/get-prebuilt-json.ts

```diff
@@ -0,0 +1,12 @@
+import fs from 'fs-extra';
+import { join } from 'path';
+
+export default async function getPrebuiltJson(directory: string) {
+  try {
+    return await fs.readJSON(join(directory, '.vercel/output/builds.json'));
```

Comment:
Turns out, the behavior of `readJSON` changes between the sync and async version of the function. The sync version will catch errors when finding the file or parsing it, but the async version only catches errors when parsing the file.
https://github.com/jprichardson/node-jsonfile/blob/master/index.js#L10-L61

It looks like they modified the behavior for the sync version without making the same change to the async version.
https://github.com/jprichardson/node-jsonfile/pull/42/files